basics/installation/system-requirements.md

```diff
@@ -11,7 +11,7 @@ PrestaShop needs the following server configuration in order to run:
 
 | Requirement | Configuration 
 | ---------------- | ------------------
-| **System** | **Unix, Linux**, Windows or MacOS.
+| **System** | **Unix**
```

Comment:
```suggestion
```

Reply:
Realistically speaking, I don't think we need a system requirement :) PrestaShop needs a web server, and its configuration is the most important. Ok, sometimes you may encounter system-specific issues, but those are not PrestaShop-related problems IMHO.